README.md

```diff
@@ -1,4 +1,4 @@
-<img src='https://redux-saga.js.org/logo/0800/Redux-Saga-Logo-Landscape.png' alt='Redux Logo Landscape' width='800px'>
+<img src='./logo/0800/Redux-Saga-Logo-Landscape.png' alt='Redux Logo Landscape' width='800px'>
```

Comment:
Could you check if this is the same image: https://redux-saga.js.org/img/Redux-Saga-Logo-Landscape.png ? I'm slightly worried that a relative path would fail to render here:  https://www.npmjs.com/package/redux-saga , so I would prefer to use an external URL here

Reply:
Yeah, they are the same image. Didn't know about that, will use the external URL then.